src/main/java/com/bandwidth/iris/sdk/model/AvailableNpaNxx.java

```diff
@@ -1,86 +0,0 @@
-package com.bandwidth.iris.sdk.model;
-
-import com.bandwidth.iris.sdk.IrisClient;
-import com.bandwidth.iris.sdk.IrisConstants;
-import com.bandwidth.iris.sdk.IrisResponse;
-import com.bandwidth.iris.sdk.utils.XmlUtils;
-
-import javax.xml.bind.annotation.XmlAccessType;
-import javax.xml.bind.annotation.XmlAccessorType;
-import javax.xml.bind.annotation.XmlElement;
-import javax.xml.bind.annotation.XmlRootElement;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-/**
- * Created by sbarstow on 11/17/14.
- */
-@XmlRootElement(name = "AvailableNpaNxx")
-@XmlAccessorType(XmlAccessType.FIELD)
-public class AvailableNpaNxx {
-
-    @XmlElement(name = "City")
-    private String city;
-    @XmlElement(name = "Npa")
-    private String npa;
-    @XmlElement(name = "Nxx")
-    private String nxx;
-    @XmlElement(name = "Quantity")
-    private int quantity;
-    @XmlElement(name = "State")
-    private int state;
-
-    public static List<AvailableNpaNxx> list(IrisClient client, Map<String, Object> query)
-            throws Exception {
```

Comment:
The pattern of having static methods for the execution code in the type was established by Steve Mitchell when the Catapult SDK development started. Every SDK, in nearly every language, follows this pattern through Catapult and Iris